docusaurus/docs/cloud/projects/settings.md

```diff
@@ -124,7 +124,7 @@ Default domain names are made of 2 randomly generated words followed by a hash.
 
 3. Click on the **Save** button.
 
-## Backups
+## Backups <CloudProBadge /> <CloudTeamBadge /> <CloudDevBadge />
```

Comment:
Maybe a small improvement suggestion: I would sort the badges in increasing price order: 
```suggestion
## Backups <CloudDevBadge /> <CloudProBadge /> <CloudTeamBadge /> 
```

Reply:
Oh yeah, this makes sense, just as seen on the Pricing page. 

Thank you